### PR TITLE
DATA-1489 - changes in order not to show changes

### PR DESCRIPTION
**Pull request by tsaridas**

### CODE REVIEW

**Comment by lhoss** [comment]
LGTM
nice  simplification of tasks, using that validate feature/attribute!
Other people also used it indeed: http://stackoverflow.com/a/28206796

File: tasks/main.yml [hunk 1]
```diff
@@ -37,30 +37,10 @@
   no_log: "{{no_log}}"
 
 - name: Create ansible sudoers in temp directory
-  template: src=etc/sudoers.d/ansible.j2 dest=/tmp/ansible
+  template: src=etc/sudoers.d/ansible.j2 dest=/etc/sudoers.d/ansible mode=0440 validate='visudo -cf %s'
```
Comment (vincepii) [inline tasks/main.yml:40]: Nice!

File: templates/etc/sudoers.d/ansible.j2 [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#{{ ansible_managed }}
-
-# Ansible-managed users with NOPASSWD
+###                                     ###
+### This file is controlled by Ansible. ###
+###                                     ###
```
Comment (vincepii) [inline templates/etc/sudoers.d/ansible.j2]: Why this change? Isn't it better to keep a variable there? Say we have it defined in our defaults, then we can do fancy things, like print the timestamp of last time this was generated, or anything we like.

Comment (tsaridas) [inline templates/etc/sudoers.d/ansible.j2]: It produces changes.

Comment (vincepii) [inline templates/etc/sudoers.d/ansible.j2]: Not if we define `ansible_managed` with static content instead of 
`ansible_managed = Ansible managed: {file} modified on %Y-%m-%d %H:%M:%S by {uid} on {host}`

**Comment by vincepii** [comment]
LGTM
